app/scrapers/ask.py

```diff
@@ -20,14 +18,13 @@ def get_page(cls,query):
         response = requests.get('http://ask.com/web', headers=header, params=payload)
         return response
 
-    @classmethod
-    def results_search(cls,query):
+    def results_search(self,query):
```

Comment:
When you do not use __self__ in a method then it is right to decorate it with [@classmethod](https://docs.python.org/3/library/functions.html#classmethod) and various linters will flag it if you do not.